ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/router-pim-sparse-mode.cfg

```diff
@@ -18,7 +19,6 @@ router pim sparse-mode
       rp address 10.238.1.161 239.12.12.17/32
```

Comment:
We already sorting it in j2 template but here you see this because of the input format.
```
router_pim_sparse_mode:
  ipv4:
    bfd: true
    rp_addresses:
      - address: 10.238.1.161
        groups:
          - 239.12.12.12/32
          - 239.12.12.13/32
          - 239.12.12.14/32
          - 239.12.12.16/32
          - 239.12.12.20/32
          - 239.12.12.21/32
        access_lists:
          - RP_ACL
          - RP_ACL2
        priority: 20
      - address: 10.238.1.161
        groups:
          - 239.12.12.17/32
        access_lists:
          - RP_ACL3
```
so here 17 comes in second element of rp_addresses

Reply:
17 is a different element of the list. So it is not easy to sort as per EOS.

Reply:
Noted!